src/base/baseTool.js

```diff
@@ -24,6 +28,13 @@ export default class {
 
     // True if tool has a custom cursor, causes the frame to render on every mouse move when the tool is active.
     this.hasCursor = false;
+
+    // Setup keybinds if present.
+    const keyBinds = this.keyBinds;
+
+    if (keyBinds) {
```

Comment:
I don't understand these five lines. Looks like we're just copying one variable into another and back again?